pkgs/development/libraries/libtransmission/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, pkgconfig
+, openssl
+, curl
+, libevent
+, inotify-tools
+, systemd
+, zlib
+, pcre
+, enableSystemd ? stdenv.isLinux
+}:
+
+let
+  version = "3.00";
+
+in stdenv.mkDerivation {
+  pname = "libtransmission";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "transmission";
+    repo = "transmission";
+    rev = version;
+    sha256 = "0ccg0km54f700x9p0jsnncnwvfnxfnxf7kcm7pcx1cj0vw78924z";
+    fetchSubmodules = true;
```

Comment:
Ideally those should be devendored but they are all from transmission, too so it is fine I think.